plans/rpmlint/README.md

```diff
@@ -7,11 +7,13 @@ Run [rpmlint] on the current Copr project or Koji build
 ## Synopsis
 
 ```yaml
-plans:
-  import:
-    url: https://github.com/packit/tmt-plans
-    ref: main
-    name: /plans/rpmlint
+discover:
+  how: fmf
+  filter: "tag: rpmlint"
+  url: https://github.com/packit/tmt-plans
+  ref: main
+execute:
+  how: tmt
```

Comment:
Just a note about hits. Do we think it makes it easier for users? I would say we should at least be consistent in this between the various checks in this repo.

Reply:
Hmm, if the user would not have a need for `rpmlintrc`, then using the `plans.import` would still be useful. The issue is that either method is good at one thing but cannot do the other:

#### `plans.import` method
- Can run context-dependent prepare steps, e.g. installing `rpminspect-data-(fedora/centos)`
- Does not have access to the caller's files

#### `discover` method
- Caller can inject files in the prepare step
- Test executable does not have access to `context`, `prepare`, etc.

Reply:
Thanks for the description. We can discuss this with tmt folks.